site.yml

```diff
@@ -10,8 +10,8 @@ content:
   - url: https://github.com/owncloud/docs.git
     branches:
     - master
+    - '10.2'
```

Comment:
That line got to be dropped, 10.2 is already part of the content definition above.

Reply:
Ah, of course. Thanks.

Reply:
@tboerger, done.